spec/system/deduplication_flow_spec.rb

```diff
@@ -40,19 +32,12 @@
       click_on 'Determination'
       click_on 'Approve and continue to disbursement'
 
-      expect(page).to have_content 'This applicant has been identified as a duplicate.'
+      expect(page).to have_content 'Potential duplicate identified'
       expect(duplicate_app.reload.approved_at).to be_nil
 
-      click_on 'Go back'
-      expect(page).to have_content 'Applicant Information'
-
-      click_on 'Determination'
-      click_on 'Approve and continue to disbursement'
-
-      expect(page).to have_content 'This applicant has been identified as a duplicate.'
       click_on 'Delete this application'
       expect(page).to have_content 'Start a new application'
       expect(AidApplication.find_by(id: duplicate_app.id)).to be_nil
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
This needs to have a system for the new behavior of the supervisor choosing to Approve or Submit after seeing the duplicate screen.